circuit/program/src/data/plaintext/to_bits.rs

```diff
@@ -49,6 +49,21 @@ impl<A: Aleo> ToBits for Plaintext<A> {
                 // Extend the vector with the bits of the struct.
                 vec.extend_from_slice(bits);
             }
+            Self::Array(elements, bits_le) => {
+                // Compute the bits of the array.
+                let bits = bits_le.get_or_init(|| {
+                    let mut bits_le = vec![Boolean::constant(true), Boolean::constant(false)]; // Variant bit.
+                    bits_le.extend(U32::constant(console::U32::new(elements.len() as u32)).to_bits_le());
+                    for value in elements {
+                        let value_bits = value.to_bits_le();
```

Comment:
as a future optimization: the allocation for this vector can be reused for all the values